src/go/k8s/controllers/redpanda/redpanda_controller.go

```diff
@@ -85,6 +85,31 @@ type RedpandaReconciler struct {
 // +kubebuilder:rbac:groups="monitoring.coreos.com",namespace=default,resources=servicemonitors,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=networking.k8s.io,namespace=default,resources=ingresses,verbs=get;list;watch;create;update;patch;delete
 
+// additional k8s resources required by flux (Cluster Scope)
+// The redpanda chart requires, for rack-awareness, that we have access to cluster scope resources.
+// So for a redpanda resource to enable rack-awareness the operator requires permissions
+// at the cluster scope.
+// +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterrolebindings,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterroles,verbs=get;list;watch;create;update;patch;delete
+
+// additional K8s resources (Cluster Scope) defined in rbac.yaml for the redpanda chart (rpk-bundle)
+// The redpanda chart requires, for rack-awareness, that we have access to cluster scope resources.
+// So for a redpanda resource to enable rack-awareness the operator requires permissions
```

Comment:
I think this comment should be removed as line 89 and 90 addressed that. This list belong to rpk-bundle, right?

Reply:
We should keep it to be clearer. Yes it is part of what is required by the rpk-bundle

Reply:
I don't understand why rpk bundle that should be used only for debug purposes needs to have cluster scope. Redpanda chart deploys (almost) everything in namespace scope. If we need to use rpk bundle should it reach to other namespaces?